### PR TITLE
[tarantool] Checkstyle updates for the Tarantool binding.

### PR DESCRIPTION
**Pull request by allanbank**

### CODE REVIEW

File: tarantool/src/main/java/com/yahoo/ycsb/db/package-info.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2014, Yahoo!, Inc. All rights reserved.
```
Comment (busbey) [inline tarantool/src/main/java/com/yahoo/ycsb/db/package-info.java]: year and attribution are incorrect.

**Comment by busbey** [comment]
+1 fine to fix copyright on package-info before merge.

**Comment by bigbes** [comment]
Seems OK

**Comment by risdenk** [comment]
Updated PR with #649

**Pull request closed without merging**